index.js

```diff
@@ -1,7 +1,25 @@
+const { gzipSizeSync } = require('gzip-size');
+
+
 addEventListener('fetch', event => {
   event.respondWith(handleRequest(event.request))
 })
 
+function getCompressionCheck(maxAllowedCompression) {
+  const baseline = gzipSizeSync('a');
+
+  function compressesTooWell(password) {
+    const size = gzipSizeSync(password);
+    return (size - baseline) < (password.length - maxAllowedCompression);
+  }
```

Comment:
I think this will be a waste of computing resources. Instead we could just put RLE here for that